### PR TITLE
docs: make a note about size/type change relationship in Redshift

### PR DESCRIPTION
**Pull request by pkutaj**
https://snowplow.zendesk.com/agent/tickets/34987

### CODE REVIEW

**Comment by pkutaj** [comment]
I'm not proposing that this is merged as suggested but I can't just open an empty PR for the sake of a discussion.

File: docs/understanding-tracking-design/versioning-your-data-structures/enterprise/index.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Different data warehouses handle schema evolution slightly differently. Use the
...
 
+* **NOTE:** In Redshift, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from say `30000` to `100000` (see [Numeric types - Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html))
```
Comment (rlh1994) [inline docs/understanding-tracking-design/versioning-your-data-structures/enterprise/index.md:21]:
```suggestion
:::caution

In Redshift, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from say `30000` to `100000` (see [Numeric types - Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html))

:::

File: docs/understanding-tracking-design/versioning-your-data-structures/enterprise/index.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ Different data warehouses handle schema evolution slightly differently. Use the
...
 
+:::caution
+
+In Redshift, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from say `30000` to `100000`. See our documentation on [how schemas translate to database types](/docs/understanding-tracking-design/json-schema-type-casting-rules/index.md), as well as Redshift’s documentation on [numeric types](https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html).
```
Comment (stanch) [inline docs/understanding-tracking-design/versioning-your-data-structures/enterprise/index.md:23]: Actually, this is true not only for Redshift, but also for Databricks.

Comment (stanch) [inline docs/understanding-tracking-design/versioning-your-data-structures/enterprise/index.md:23]:
```suggestion
In Redshift and Databricks, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from `30000` to `100000`. See our documentation on [how schemas translate to database types](/docs/understanding-tracking-design/json-schema-type-casting-rules/index.md).
```

**Review by stanch — APPROVED** [review]

**Pull request merged**